aptos-move/framework/aptos-framework/sources/vesting.move

```diff
@@ -416,8 +416,12 @@ module aptos_framework::vesting {
         assert_active_vesting_contract(contract_address);
 
         let vesting_contract = borrow_global_mut<VestingContract>(contract_address);
-        let contract_signer = &get_vesting_account_signer_internal(vesting_contract);
-        staking_contract::unlock_rewards(contract_signer, vesting_contract.staking.operator);
+        let operator = vesting_contract.staking.operator;
+        let (total_active_stake, _, commission_amount) =
+            staking_contract::staking_contract_amounts(contract_address, operator);
+        // Accumulated rewards, excluding unpaid commission, that entirely belongs to the shareholders.
```

Comment:
I think the comment could be clearer about the difference between this and the accumulated rewards returned from the staking_contract_amounts call